addon/ng2/blueprints/ng2/files/karma.conf.js

```diff
@@ -14,6 +15,8 @@ module.exports = function(config) {
       {pattern: 'node_modules/angular2/bundles/http.dev.js', included: true, watched: true},
       {pattern: 'node_modules/angular2/bundles/router.dev.js', included: true, watched: true},
       {pattern: 'node_modules/angular2/bundles/testing.dev.js', included: true, watched: true},
+      {pattern: 'node_modules/es6-shim/es6-shim.js', included: true, watched: true},
```

Comment:
Shouldn't these be in the same order as https://github.com/angular/angular-cli/blob/master/lib/broccoli/angular2-app.js#L21-L30 ?